From 6370896cbb39d53209daca4c01e91a05a9c1a49e Mon Sep 17 00:00:00 2001
From: CaoSong <caosong@mprc.pku.edu.cn>
Date: Fri, 2 Dec 2011 18:08:10 +0800
Subject: [PATCH 114/641] UniCore64: Modify arch hwdef-cpu.h

Modify the block comment and add the programlisting label for the align.
Signed-off-by: Cao Song <caosong@mprc.pku.edu.cn>
---
 arch/unicore64/include/arch/hwdef-cpu.h |   14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/arch/unicore64/include/arch/hwdef-cpu.h b/arch/unicore64/include/arch/hwdef-cpu.h
index 0877e48..b2e086f 100644
--- a/arch/unicore64/include/arch/hwdef-cpu.h
+++ b/arch/unicore64/include/arch/hwdef-cpu.h
@@ -5,17 +5,17 @@
  * DOC: HWDEF_CPU_H
  * This file defines the bit pattern for the processor status registers
  * and the processor flag registers.
- */
-
-/**
+ *
+ * \\\\lt:programlisting\\\\gt:
+ * -
  * ASR_MODE_MASK:	mode state mask
  * ASR_MODE_USER:	user state
  * ASR_MODE_PRIV:	privilege state
  * ASR_MODE_DEBUG:	debug state
- *
+ * -
  * ASR_T_BIT:		multi processors consistency enable bit
  * ASR_D_BIT:		debug mode enable bit
- *
+ * -
  * ASR_INTR_MASK:	interrupt mask
  * ASR_INTR_SMP:	interrupt between multi processors
  * ASR_INTR_LSU:	interrupt about LSU
@@ -23,12 +23,14 @@
  * ASR_INTR_OTM:	outer timer interrupt
  * ASR_INTR_DEV:	outer device interrupt
  * ASR_INTR_PFM:	performance monitor interrupt
- *
+ * -
  * AFR_MASK:		AFR flag bit mask
  * AFR_V_BIT:		overflow flag bit
  * AFR_C_BIT:		carry flag bit
  * AFR_Z_BIT:		zero flag bit
  * AFR_S_BIT:		negative flag bit
+ * -
+ * \\\\lt:/programlisting\\\\gt:
  */
 
 #include <arch/bitfield.h>
-- 
1.7.9.5

